packages/neon_framework/lib/src/pages/login_flow.dart

```diff
@@ -39,10 +38,7 @@ class _LoginFlowPageState extends State<LoginFlowPage> {
 
     initSubscription = bloc.init.listen((result) async {
       if (result.hasData) {
-        await launchUrlString(
-          result.requireData.login,
-          mode: LaunchMode.externalApplication,
```

Comment:
Why did you change the launch mode?
My last information was that it is preferred to use the browser, as webview might have issues with proxy setups

Reply:
I forgot to change it

Reply:
Still the same.

Reply:
No, the method that wraps it always uses LaunchMode.externalApplication